test/util/VariableNameCreatorTest.ts

```diff
@@ -39,4 +48,57 @@ describe("VariableNameCreator", function() {
 				[ "asdYeyYoo", "yeyYoo", "yoo" ], "asd.yey.yoo"),
 			"oAsdYeyYoo");
 	});
-});
\ No newline at end of file
+
+	it("getUniqueParameterName", function() {
+		assert.strictEqual(
```

Comment:
Maybe add assertion texts for better understanding of the tests

Reply:
ok